src/main/java/org/jabref/gui/Base.css

```diff
@@ -1734,9 +1734,41 @@ We want to have a look that matches our icons in the tool-bar */
     -fx-effect: dropshadow(three-pass-box, rgba(0, 0, 0, 0.6), 8, 0.0, 0, 0);
 }
 
+.welcome-label {
+    -fx-font-size: 40px;
+    -fx-text-fill: -jr-theme-text;
+    -fx-font-family: "Arial";
+}
+
+.description-label {
```

Comment:
Please use a more explizit selector name for the CSS class, as this may interfere with other places similar or the same classes are used. Use welcome-* like above and ahead as a prefix.